qubes/vm/adminvm.py

```diff
@@ -98,7 +98,7 @@ def __str__(self):
         return self.name
 
     def __lt__(self, other: object):
-        if not isinstance(other, BaseVM):
+        if not isinstance(other, LocalVM):
```

Comment:
This should remain BaseVM, otherwise sorting VM objects will not work with RemoteVM